### DISCUSSION TITLE
Capitalize Widget Text At A Glance

### DISCUSSION THREAD

**Discussion by TheJNXx** [LawnchairLauncher/lawnchair#2967] [Ideas]
[Screenshot_20220919-141548_Lawnchair.jpg](https://user-images.githubusercontent.com/108956098/191097516-149d0218-b871-4001-aa17-c1939c8bad53.jpg)

**Comment by SuperDragonXD** [comment]
This is more of an issue with the translation of the text (since in English, it's capitalized). If you want to fix it yourself, please see the [translation instructions in the README file](https://github.com/LawnchairLauncher/lawnchair#translate).
